src/test/java/edu/harvard/iq/dataverse/dataverse/validation/BannerErrorHandlerTest.java

```diff
@@ -0,0 +1,152 @@
+package edu.harvard.iq.dataverse.dataverse.validation;
+
+import com.google.common.collect.Sets;
+import edu.harvard.iq.dataverse.dataverse.banners.BannerLimits;
+import edu.harvard.iq.dataverse.dataverse.banners.DataverseBanner;
+import edu.harvard.iq.dataverse.dataverse.banners.DataverseLocalizedBanner;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.omnifaces.util.Faces;
+
+import javax.faces.application.FacesMessage;
+import javax.faces.context.ExternalContext;
+import javax.faces.context.FacesContext;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Locale;
+
+import static edu.harvard.iq.dataverse.util.DateUtil.convertToDate;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BannerErrorHandlerTest {
+
+    @Mock
+    private FacesContext facesContextMock;
+    @Mock
+    private ExternalContext externalContextMock;
+    @Mock
+    private Locale locale;
+    @Captor
+    private ArgumentCaptor<FacesMessage> facesMesssage;
+
+    private static final Path BANNER_PATH = Paths.get("src/test/resources/images/banner.png");
+    private static final java.util.Date FROM_TIME = convertToDate(
+            LocalDateTime.of(2018, 12, 1, 9, 15, 45));
+    private static final java.util.Date TO_TIME = convertToDate(
+            LocalDateTime.of(2018, 11, 2, 10, 25, 55));
+
+    @Before
+    public void setup() {
+        // do some wiring
+        Faces.setContext(facesContextMock);
+
+        Mockito.when(facesContextMock.getExternalContext()).thenReturn(externalContextMock);
+        Mockito.when(facesContextMock.getExternalContext().getRequestLocale()).thenReturn(locale);
```

Comment:
Why not just make `when(externalContextMock.getRequestLocale()).then...`? You already have that as class field.